lib/php/src/ext/thrift_protocol/config.m4

```diff
@@ -25,6 +25,8 @@ PHP_ARG_ENABLE(thrift_protocol, whether to enable the thrift_protocol extension,
 
 if test "$PHP_THRIFT_PROTOCOL" != "no"; then
   PHP_REQUIRE_CXX()
+  PHP_ADD_LIBRARY_WITH_PATH(stdc++, "", THRIFT_PROTOCOL_SHARED_LIBADD)
+  PHP_SUBST(THRIFT_PROTOCOL_SHARED_LIBADD)
   CXXFLAGS="$CXXFLAGS -std=c++11"
```

Comment:
Interesting - the project does not require c++11, but apparently here it does. :|